workflow/src/main/kotlin/com/r3/corda/sdk/token/workflow/flows/UpdateEvolvableTokenResponder.kt

```diff
@@ -25,6 +25,6 @@ class UpdateEvolvableTokenResponder(val otherSession: FlowSession) : FlowLogic<S
         }
 
         // Resolve the creation transaction.
-        return subFlow(ReceiveFinalityFlow(otherSideSession = otherSession, statesToRecord = StatesToRecord.ONLY_RELEVANT))
+        return subFlow(ReceiveFinalityFlow(otherSideSession = otherSession, statesToRecord = StatesToRecord.ALL_VISIBLE))
```

Comment:
This change is because, before, the other participants were not recording the updates? I think it was only relevant before because there was only one maintainer but now that's not the case. Good spot.

Reply:
Yeah, it was failing some tests.